geniza/corpus/tests/test_corpus_models.py

```diff
@@ -211,6 +213,24 @@ def test_str(self):
         doctype.display_label = "Legal document"
         assert str(doctype) == "Legal document"
 
+    def test_natural_key(self):
+        """Should use name as natural key"""
+        doc_type = DocumentType(name="SomeType")
+        assert len(doc_type.natural_key()) == 1
+        assert "SomeType" in doc_type.natural_key()
+        doc = Document.objects.create()
+        doc.doctype = doc_type
+        serialized_doc = serialize(
+            "json", [doc], use_natural_foreign_keys=True, use_natural_primary_keys=True
+        )
+        assert '"doctype": ["SomeType"]' in serialized_doc
```

Comment:
this part seems unnecessary to me  — you're testing django logic, not local code